src/views/new-evaluation/new-evaluation.js

```diff
@@ -0,0 +1,590 @@
+    ${NewEvaluationForm()} 
+    ${Footer()} 
+    `;
+}
+
+function NAVNewEvaluations (options = {}) {
+    return `
+    <header class="header">
+        <img class="logo" src="assets/logo-v2.png">
+        <a class="menu_button" href="evaluations.html">Evaluations</a>
+        <a class="menu_button current_menu_button" href="new-evaluation.html">New Evaluation</a>
+        <a class="menu_button" href="login.html" style="float: right">Logout</a>
+    </header>
+    `;
+}
+
+function CandidateDetailsForm (options = {}) {
+    return `
+    <div>
+        <input class="evaluation_form_header" type="text"  name="Candidate" placeholder="Candidate">
+        <input class="evaluation_form_header" type="text" name="Interviewer" placeholder="Interviewer">
+        <input class="evaluation_form_header" type="date" name="Date" value="04/22/2012">
+    </div>
+    `;
+}
+
+function TechnicalLevelPicker (option = {}) {
+    return `
+    <h2 class="evaluation_category">Technical level</h2>
+        <table class="evaluation_form_table">
+            <tr class="evaluation_form_table_header skin_evaluation_form_table_header">
+                <th>Trainee</th>
+                <th>Junior</th>
+                <th>Middle</th>
+                <th>Senior</th>
+            </tr>
+            <tr class="evaluation_form_table_content">
+                <td>
+                    <input type="radio" name="level" value="Trainee"></td>
+                <td>
+                    <input type="radio" name="level" value="Junior1">
+                    <input type="radio" name="level" value="Junior2">
+                    <input type="radio" name="level" value="Junior3">
+                </td>
+                   <td>
+                    <input type="radio" name="level" value="Middle1">
+                    <input type="radio" name="level" value="Middle2">
+                    <input type="radio" name="level" value="Middle3">
+                   </td>
+                    <td>
+                    <input type="radio" name="level" value="Senior1">
+                    <input type="radio" name="level" value="Senior2">
+                    <input type="radio" name="level" value="Senior3">
+                </td>
+            </tr>
+        </table>
+    `;
+}
+
+function Textarea (options = {}) {
+    return `
+    <h2 class="evaluation_category">${options.label}</h2>
+    <textarea class="evaluation_comments" rows="5" cols="132" placeholder="${options.placeholder}"></textarea>
+    `;
+}
+
+function NewEvaluationForm (option = {}) {
+    return `
+    <div class="evaluation_form">
+    <form method="post" action="action.php">
+        ${CandidateDetailsForm()}
+        ${TechnicalLevelPicker()}
+        ${Textarea()}
+        ${Textarea()}
+        ${Textarea()}
+        <fieldset class="evaluation_options">
+            <legend class="evaluation_category">OOP, Design Patterns</legend>
+            <ul>
+                <li>Classes</li>
+                <li>
```

Comment:
What a big bunch of highly repetitive code here... If only there were ways of doing this so we didn't have to repeat ourselves all over the place... :stuck_out_tongue:

Reply:
Yep, I still have to take care of this and generate the elements using JS. Haven't had the time to finish it yesterday.